main/design_documents/ARM_NEON_SVE_BRIDGE.rst

```diff
@@ -0,0 +1,78 @@
+=======================================
+Design Document for ARM_NEON_SVE_BRIDGE
+=======================================
+
+This document describes the design of the intrinsics defined in
+`NEON_SVE Bridge <../acle.rst#neon-sve-bridge>`_. These C intrinsics enable the
+conversion between NEON and SVE vectors with no performance overhead, and
+duplicating NEON vectors across SVE vectors.
+
+Location
+########
+
+The intrinsics are defined in a header file called 
+``arm_neon_sve_bridge.h``.
+
+This header file includes ``arm_neon.h`` and ``arm_sve.h``.
+It also defines ``__ARM_NEON_SVE_BRIDGE`` as 1.
```

Comment:
Would need to be updated if we do change how this macro is defined.

Reply:
done.